### ISSUE TITLE
Too complicated.

### ISSUE THREAD

**Issue by scottonanski** [glidejs/glide#633]
I downloaded it, and unzipped it...

Have no idea what I'm looking at. It's just a bunch of files.

Try making it less complicated.  It might gain more traction.

**Comment by hobyvh** [comment]
I agree. This was made assuming that no one wants to install it without processing through NPM to generate the final js and css files. The rest of it may be made without dependencies but NPM is required to even get the files we'd deploy.

**Comment by jonXmack** [comment]
> I agree. This was made assuming that no one wants to install it without processing through NPM to generate the final js and css files. The rest of it may be made without dependencies but NPM is required to even get the files we'd deploy.

There's a big download button on the [Setup page](https://glidejs.com/docs/setup#download) as well as instructions on how to [structure your code](https://glidejs.com/docs/setup#structure)

**Comment by vurso** [comment]
Agreed I hit the download button downloaded the source did a build no dist folder present nothing much happens.  In the end I used NPM to install the package then retrieve the files and pull them into my main project (where I can't/won't use NPM).

**Comment by vurso** [comment]
Agreed instructions all seem to want users to go with the npm package but not everyone does and there build instructions could be present on the readme so developers know immediately how to build it locally and produce the necessary JS file(s) for inclusion in their "non App projects".

**Comment by eugeneandrienko** [comment]
> There's a big download button on the [Setup page](https://glidejs.com/docs/setup#download) as well as instructions on how to [structure your code](https://glidejs.com/docs/setup#structure)

That big download button — do not links to release. It downloads just sources — there are no ``glide.min.js`` or ``glide.core.min.css`` files, which mentioned in instructions.

Fortunately, these files can be downloaded from unpkg, with slightly modified link from the same [Setup page](https://glidejs.com/docs/setup#download): https://unpkg.com/browse/@glidejs/glide@3.6.0/dist/
